app/jobs/bulk_action_jobs.rb

```diff
@@ -213,18 +213,30 @@ def perform(collection_id, overwrite = true, save_field = nil)
           media_object.lending_period = collection.default_lending_period
         end
 
+        # If MediaObject visibility is different than Collection, the collection visibility
```

Comment:
Since these variables are only used in the special access case, what do you think about moving the `if save_field == "special_access"` conditional to wrap these?  That would also DRY up the if and elsif clauses below.